api/c/indigo/src/indigo_internal.h

```diff
@@ -200,6 +200,8 @@ class DLLEXPORT IndigoObject
     virtual MonomersProperties& getMonomersProperties();
     virtual void copyProperties(PropertiesMap&);
 
+    void toBase64String(Array<char>& str);
```

Comment:
do we use this function anywhere? if yes, need to change to return result array instead of converting input to base64